daemon/daemon.go

```diff
@@ -977,7 +826,7 @@ func (daemon *Daemon) Mount(container *Container) error {
 	}
 	if container.basefs == "" {
 		container.basefs = dir
-	} else if container.basefs != dir {
+	} else if container.basefs != dir && runtime.GOOS != "windows" {
```

Comment:
This was actually an older version of the method that got pulled in by mistake.  The updated version is in the latest diff.  I wanted to leave the error case for Linux (graph driver returning inconsistent paths).  On Windows, there are circumstances where the mounted volume path for a given disk chain can change, and these are expected, not errors.  So on Windows, we instead want to accept whatever path is returned by the graph driver.